cli.js

```diff
@@ -12,7 +13,23 @@ const cli = cac('create-nuxt-app')
 cli
   .command('[out-dir] [options]', 'Generate in a custom directory or current directory')
   .option('--edge', 'To install `nuxt-edge` instead of `nuxt`')
-  .action((outDir = '.') => {
+  .option('--info', 'Print out debugging information relating to the local environment')
+  .action(async (outDir = '.') => {
+    const hasInfoArg = process.argv.slice(2)[0] !== '--edge'
+    if (hasInfoArg) {
```

Comment:
Maybe we could extract the if body as a function?

Reply:
@manniL What you intend is to have the implementation within a dedicated method?

Reply:
Improving code readability :relaxed: